commit e1e5b29add5eb6e07711b2a96b1935e63e1e80fe
Author: Alexandre Belloni <alexandre.belloni@free-electrons.com>
Date:   Fri Mar 17 00:48:33 2017 +0000

    usb: gadget: udc: atmel: remove memory leak
    
    [ Upstream commit 32856eea7bf75dfb99b955ada6e147f553a11366 ]
    
    Commit bbe097f092b0 ("usb: gadget: udc: atmel: fix endpoint name")
    introduced a memory leak when unbinding the driver. The endpoint names
    would not be freed. Solve that by including the name as a string in struct
    usba_ep so it is freed when the endpoint is.
    
    Signed-off-by: Alexandre Belloni <alexandre.belloni@free-electrons.com>
    Signed-off-by: Felipe Balbi <felipe.balbi@linux.intel.com>
    Signed-off-by: Sasha Levin <alexander.levin@verizon.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1450_e1e5b29add5eb6e07711b2a96b1935e63e1e80fe.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1450_e1e5b29add5eb6e07711b2a96b1935e63e1e80fe.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
